modules/ags/src/main/scala/lucuma/ags/Ags.scala

```diff
@@ -108,9 +108,11 @@ object Ags {
     // This is essentially a cache of geometries avoiding calculatting them
     // over and over again as they don't change for different positions
     val calcs       = params.posCalculations(List(position))
+    // use the slowest speed to filter out
+    val bc          = guideSpeeds.find(_._1 === GuideSpeed.Slow).map(_._2)
 
     in =>
-      in.map { gsc =>
+      in.filter(c => c.gBrightness.exists(g => bc.exists(_.contains(Band.Gaia, g)))).map { gsc =>
```

Comment:
This looks good.  I'm just wondering whether we get the full range of brightnesses with just the slowest guide speed.

Reply:
I'm probably wrong but it seemed like it would need the saturation limit from the `Fast` guide speed and the faint limit from the `Slow` guide speed.

Reply:
Ok, if there is no saturation limit then, :+1: